docs/METRICS.md

```diff
@@ -183,4 +187,17 @@ If this value is beyond the alarm intervals an alarm should have been sent.
 The value represents the effective remaining repair time for the table to be fully repaired in milliseconds.
 This is the time ecChronos will have to wait for Cassandra to perform repair,
 this is an estimation based on the last repair of the table.
-The value should be `0` if there is no repair ongoing for this table.
\ No newline at end of file
+The value should be `0` if there is no repair ongoing for this table.
+
+| t          | count | mean_rate | m1_rate  | m5_rate  | m15_rate | rate_unit     |
+|------------|-------|-----------|----------|----------|----------|---------------|
+| 1660042648 | 0     | 0.000000  | 0.000000 | 0.000000 | 0.000000 | events/second |
```

Comment:
This is valuable information. Perhaps we should also add a counter for succeeded RepairTasks.